tools/testbench/file.c

```diff
@@ -837,9 +838,10 @@ static int file_prepare(struct comp_dev *dev)
 
 	/* set downstream buffer size */
 	stream = &buffer->stream;
+	samples = 2 * periods * dev->frames * stream->channels;
```

Comment:
looks like before it was 2 for 16 bits and 4 for 32 bits, now you use `sizeof(intNN_t)` explicitly, which is good, but what does this 2 now mean here?

Reply:
It's a hack, the proper place to fix is in topology. I'll update soon this PR.

Reply:
Now the periods from topology is preserved, no multipliers any more here. If issues, the test topology buffer size needs fix.